src/Analyzers/CSharp/CodeFixes/GenerateConstructor/CSharpGenerateConstructorService.cs

```diff
@@ -45,7 +43,7 @@ protected override bool TryInitializeConstructorInitializerGeneration(
         SyntaxNode node,
         CancellationToken cancellationToken,
         out SyntaxToken token,
-        out ImmutableArray<Argument> arguments,
+        out ImmutableArray<Argument<ExpressionSyntax>> arguments,
```

Comment:
the 'Argument' type was moved out of a specific feature and placed in the 'CodeGeneration' namespace.  It was previously using the TExpressionSyntax type from its container.  But as a new top level type, it had to become generic, and this is the fallout.